crates/primitives/fee/src/lib.rs

```diff
@@ -33,17 +33,16 @@ pub const FEE_TRANSFER_N_STORAGE_CHANGES: u8 = 2; // Sender and sequencer balanc
 /// Number of storage updates to actually charge for the fee transfer tx.
 pub const FEE_TRANSFER_N_STORAGE_CHANGES_TO_CHARGE: u8 = FEE_TRANSFER_N_STORAGE_CHANGES - 1; // Exclude the sequencer balance update, since it's charged once throughout the batch.
 
-// TODO: add real values here.
 // FIXME: https://github.com/keep-starknet-strange/madara/issues/330
```

Comment:
does it make sense to keep this comment here? cc @tdelabro

Reply:
We can remove it

Reply:
And the next line too I belive